BUILD.bpf

```diff
@@ -0,0 +1,24 @@
+COPTS = [
+    '-Iexternal/libbpf/include',
+    '-Iexternal/libbpf/include/uapi',
+    '-DCOMPAT_NEED_REALLOCARRAY',
+    '-fPIC',
+    '-fvisibility=hidden',
+    '-g',
+    '-O2',
+    '-Werror',
+    '-Wall',
+    '-c',
```

Comment:
Does it compile if you remove -c, -fPIC, -g and -O2? according to the docs, copts _adds_ flags to the build process, and -fPIC is for position independent code (.so library), -c to create objects, -g to enable debugging, and -O2 for optimization. All flags bazel should provide already / automatically depending on the target.

Reply:
Earlier it had thrown an error but it seems to work without -c, -fPIC, -g and -O2? 
Will remove them.